### PR TITLE
Russian engage pages

### PR DESCRIPTION
**Pull request by sowasred2012**
## Done

- Enabled display of cyrillic characters
- Added Russian ("Русский") as an option in the /engage index

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/engage
    - Be sure to test on mobile, tablet and desktop screen sizes
- Select "Русский" from the language dropdown, there should be four results
- Click any of them, the font on the engage page should look a lot better than it does on the live site, and the font family being used should be "UbuntuFullCharSet".
- Go to a non Russian engage page, inspect the text, the font family should be "Ubuntu"

## Screenshots

Before:

Screenshot from 2022-02-07 16-59-54

After:

Screenshot from 2022-02-07 17-00-06

### CODE REVIEW

File: templates/engage/base.html [hunk 1]
```diff
@@ -16,12 +16,37 @@
   {% if document['metadata']['path'] == "/engage/ShellOnboarding" %}
     <meta name="robots" content="noindex, nofollow" />
   {% endif %}
+
+  {# The Ubuntu font used acrros the site doesn't contain Russian characters, #}
```
Comment (anthonydillon) [inline templates/engage/base.html:20]:
```suggestion
  {# The Ubuntu font used across the site doesn't contain Russian characters, #}
```

File: templates/engage/base.html [hunk 2]
```diff
@@ -16,12 +16,37 @@
...
+  {# import the full char set for any Russian engage pages #}
```
Comment (anthonydillon) [inline templates/engage/base.html:21]:
```suggestion
  {# if the page content is Russian then import the full character set #}
```

**Review by anthonydillon — CHANGES_REQUESTED** [review]
It's a shame that we need to load so many font's. Would need to explore some ideas to work out what might be better but this works.

My concern is that the text on this page would not render nicely? https://ubuntu-com-11230.demos.haus/engage?language=ru

**Comment by anthonydillon** [comment]
In fact I think you can just use:
```
@font-face {
          font-display: $font-display-option;
          font-family: 'Ubuntu';
          font-style: normal;
          font-weight: $font-weight-regular-text;
          src: url('#{$assets-path}5bea8279-Ubuntu-L-cyrillic-subset.woff2') format('woff2'), url('#{$assets-path}b8058442-Ubuntu-L-cyrillic-subset.woff') format('woff');
          unicode-range: U+0400-045F, U+0490-0491, U+04B0-04B1, U+2116;
        }
```

**Comment by anthonydillon** [comment]
Or just set the Vanilla setting `$font-allow-cyrillic-greek-latin` to true.

**Comment by anthonydillon** [comment]
The `Ubuntu-L-cyrillic-ext-subset.woff2` file is 26kb where as the `Ubuntu-L_W.woff2` you are using is 127.3kb,

**Comment by sowasred2012** [comment]
@anthonydillon I just tried that subset, it _mostly_ works, but the heading in the hero strip uses `font-weight: 100;`, which that subset doesn't seem to cover - @pmahnke is going to create a new subset for that font-weight to use here.

**Comment by pmahnke** [comment]
I created these two subsets:

- [light](https://assets.ubuntu.com/v1/3026c62c-Ubuntu-L-subset-cyrillic.woff2)
- [regular](https://assets.ubuntu.com/v1/d72414f8-Ubuntu-R-subset-cyrillic.woff2)

**Comment by anthonydillon** [comment]
> My concern is that the text on this page would not render nicely? https://ubuntu-com-11230.demos.haus/engage?language=ru

This is still a concern

**Comment by anthonydillon** [comment]
Sorry @sowasred2012 I miss read the code in Vanilla. You can use both settings together:
```sass
$font-use-subset-latin: true;
$font-allow-cyrillic-greek-latin: true;
```
Looks like we just need to add light font to the set provided by Vanilla. Which @pmahnke is doing, right?

Then you can remove the `UbtuOverride` stuff. As we `$font-display-option` variable locally on ubuntu.com. This slightly increases (138kb to 197kb) the font size across the site but if we have Russian on the site we should include the font globally.

**Comment by carkod** [comment]
Wow looks nice, no difference between the two alphabets, but you may also want to include it in the /index page.

index-engage-cyrllic

**Comment by sowasred2012** [comment]
> Update: Oh I suppose that is the issue with the font weight variations?
Yep, will be looking at this again today.

File: static/sass/styles.scss [hunk 3]
```diff
@@ -10,23 +10,11 @@
...
-    url("https://assets.ubuntu.com/v1/8619add2-Ubuntu-L_W.woff") format("woff");
-}
-
 // import vanilla-framework
 @import "vanilla-framework/scss/build";
 @import "vanilla-placeholders";
 
-// Vanilla framework overrides
-$font-base-family: '"UbtuOverride", -apple-system, "Segoe UI", "Roboto", "Oxygen", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif' !default;
+// // Vanilla framework overrides
```
Comment (anthonydillon) [inline static/sass/styles.scss:17]: Do you need this comment anymore? It's double commented too.

**Review by anthonydillon — APPROVED** [review]
Piece of art @sowasred2012, perfect :+1:

File: static/sass/styles.scss (hunk 3 above)
Comment (sowasred2012) [reply static/sass/styles.scss]: Nope!

**Pull request merged**